widgets/src/themes.rs

```diff
@@ -1,34 +1,2 @@
 mod styles_sheet;
-use ribir_core::{fill_svgs, prelude::*};
 pub use styles_sheet::*;
-
-pub(crate) fn add_to_theme(theme: &mut FullTheme) {
-  fill_svgs! {
-    theme.icon_theme,
-    svgs::ADD: "./themes/icons/add_FILL0_wght400_GRAD0_opsz48.svg",
```

Comment:
remove here, if developer don't use material theme, it only use default theme, it will not found svg file.

Reply:
I'll move these icons into core.